components/script/dom/htmlinputelement.rs

```diff
@@ -937,7 +909,7 @@ impl HTMLInputElement {
             _ => ()
         }
 
-        self.SetValue(self.DefaultValue())
+        self.update_text_contents(self.DefaultValue(), false)
```

Comment:
We should probably pass this value in to `reset` as an argument. I'm pretty sure there are places that call reset that don't want this behaviour.

Reply:
It works this way, infact there is just one `reset` call in `htmlinputelement.rs`